modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2070,4 +2071,114 @@ describe('V2 Wallet:', function () {
       });
     });
   });
+
+  describe('unspents APIs with string type minValue and maxValue', function () {
+    // using the same test for fanout and consolidate unspent API
+    it('consolidateUnspents should handle string type minValue and maxValue', async function () {
+      const params = { minValue: '123', maxValue: '9000000000000000000123' };
+
+      const scope = nock(bgUrl)
+        .post(`/api/v2/${wallet.coin()}/wallet/${wallet.id()}/consolidateUnspents`, body => {
+          return typeof body.minValue === 'string' && body.minValue === '123'
+              && typeof body.maxValue === 'string' && body.maxValue === '9000000000000000000123';
+        })
```

Comment:
This is very repetitive. Please add a method that returns a nock.

Reply:
unresolved

Reply:
resolved by creating common function matchMinMaxValues